src/components/mdInputContainer/common.js

```diff
@@ -29,6 +39,15 @@ export default {
       this.parentContainer.enableCounter = this.maxlength > 0;
       this.parentContainer.counterLength = this.maxlength;
     },
+    lazyEventEmitter() {
```

Comment:
I'm not sure why is github showing like that, but if you check the raw code, you'll see it is in methods. I don't know what sorcery happened here.